src/driver/sap/SapDriver.ts

```diff
@@ -493,9 +500,7 @@ export class SapDriver implements Driver {
 
         if (value === null || value === undefined) return value
 
-        if (columnMetadata.type === Boolean) {
-            return value === true ? 1 : 0
```

Comment:
Is it intentional to lose the boolean conversions here?

Reply:
I think it was copy-pasted from a driver that doesn't support boolean (mssql?)

But `@sap/hana-client` is okay with receiving `boolean` values, no explicit conversion to 0 & 1 is needed.